packages/graphql-transformer-common/src/ResourceConstants.ts

```diff
@@ -2,6 +2,16 @@ export class ResourceConstants {
 
     public static NONE = "NONE"
 
+    public static readonly ENVIRONMENT_CONTEXT_KEYS = {
```

Comment:
Can we move this to the new package and use it instead of the ResourceConstants.PARAMETERS?

Reply:
done